### PR TITLE
Getting started edits

### PR DESCRIPTION
**Pull request by JuLeeAtPlanet**
WIP that does not break build. Reorg of pages and initial info arch: good checkin point.

### CODE REVIEW

File: docs/cli/cli-reference.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<<<<<<< HEAD
```
Comment (cholmes) [inline docs/cli/cli-reference.md:1]: Looks like some github diff stuff snuck in here... Would be good to remove it before merging.

File: docs/get-started/quick-start-guide.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
+<<<<<<< HEAD
```
Comment (cholmes) [inline docs/get-started/quick-start-guide.md:1]: Another git merge conflict that didn't get resolved.

File: docs/get-started/quick-start-guide.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+Planet's APIs require an account for use. For more information on where to find your API key, or get one if you don't already have one, see [Get Your API Key](get-your-api-key.md).
+>>>>>>> 2fa3e4e (first docs reorg)
+
+## Step 4: Check the toolkit version
+
+```console
+$ planet --version
+```
+
+<<<<<<< HEAD
```
Comment (cholmes) [inline docs/get-started/quick-start-guide.md:49]: Also here, conflict to resolve

File: docs/get-started/quick-start-guide.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+[TOC]
+
+## Step 1: Activate a virtual environment
+
+This is a Python toolkit, so you'll need to install Python (version 3.7 or greater), and setup and install a virtual environment.
+
+Yep. Even if you're not writing code—and only using the "no code" CLI part of the Toolkit—you're using Python to communicate with the Planet Labs PBC servers. It's not too tricky, but it does require a presence of mind to complete. If you need help with Python install and setting up a virtual environment, read [Getting started with venv & Planet SDK](venv-tutorial.md).
+>>>>>>> 2fa3e4e (first docs reorg)
+
+## Step 2: Install the Planet Earth Observation Toolkit
```
Comment (cholmes) [inline docs/get-started/quick-start-guide.md:24]: Where did this name come from?

File: docs/get-started/quick-start-guide.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+```console
+$ pip install planet --user
+```
+
+The [--user](https://pip.pypa.io/en/stable/user_guide/#user-installs)
+flag is highly recommended for those new to pip.
+
+## Step 3: Get your API Key ready
+
+<<<<<<< HEAD
```
Comment (cholmes) [inline docs/get-started/quick-start-guide.md:37]: Another merge conflict

File: docs/get-started/quick-start-guide.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+## Step 7: Creating your first order
+
+[*****TBW*****]
+
+The Planet SDK for Python allows Python developers to write software that makes
+use of the following Planet APIs:
+
+* [orders](https://developers.planet.com/docs/orders/)
+* [data](https://developers.planet.com/docs/data/) (not implemented)
```
Comment (cholmes) [inline docs/get-started/quick-start-guide.md:81]:
```suggestion
* [data](https://developers.planet.com/docs/data/) 
```

This is implemented now - have been meaning to get this in, and I think this PR is fine to fix - I realize it's not a problem with this PR.

File: docs/get-started/quick-start-guide.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+* [analytics](https://developers.planet.com/docs/analytics/) (not implemented)
+* [basemaps](https://developers.planet.com/docs/basemaps/) (referred to in the client as `mosaics`) (not implemented)
```
Comment (cholmes) [inline docs/get-started/quick-start-guide.md:83]:
```suggestion
* [basemaps](https://developers.planet.com/docs/basemaps/) (referred to in the V1 client as `mosaics`) (not implemented)
```

File: docs/get-started/quick-start-guide.md [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+
+The client modules within the Python library are asynchronous, which greatly
+speeds up many interactions with Planet's APIs. Support for asynchronous
+development is native to Python 3.6+ via the
+[`asyncio` module](https://docs.python.org/3/library/asyncio.html). A great
+resource for getting started with asynchronous programming in Python is
+https://project-awesome.org/timofurrer/awesome-asyncio. The Writings and Talks
```
Comment (cholmes) [inline docs/get-started/quick-start-guide.md:90]: Yes! @jreiberkyle and I were just talking about putting in a link to some nice intro to async.

File: docs/get-started/upgrading.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 # Upgrading
 
-The Planet SDK for Python is Version 2 of what was previously referred to as
+The Planet Earth Observation Toolkit is Version 2 of what was previously referred to as
```
Comment (cholmes) [inline docs/get-started/upgrading.md:3]: I see we're at least consistent on this, so that's good, but where was the decision about this made?

**Review by cholmes — APPROVED** [review]
I have a number of review comments, particularly around unresolved merge conflicts, so do try to fix those before merging. But marking as 'approve' so you can get it in - I agree it'd be good to get these on 'main' even though it's WIP.

File: custom_theme/home.html [hunk 10]
```diff
@@ -0,0 +1,45 @@
+{% extends "main.html" %}
```
Comment (jreiberkyle) [inline custom_theme/home.html:1]: Could this folder go in the docs subfolder?

File: docs/get-started/quick-start-guide.md (hunk 7 above)
Comment (jreiberkyle) [reply docs/get-started/quick-start-guide.md:83]: It seems we can remove references to what V1 called things everywhere except for the upgrading section.

File: design-docs/content-gsx-notes.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
+# Create a Getting Started Experience (GSX)
+
+We'd like to make sure folks who come to the docs site can get started easily. We'd like them to know how to:
+
+* Get an API key
+* Authenticate against the service
+* Search and download data with the cURL CLI
```
Comment (jreiberkyle) [inline design-docs/content-gsx-notes.md:7]: why is cURL included here?

File: design-docs/content-gsx-notes.md [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+
+# A clear top nav will help with the GSX
+
+## Add a clear top nav to mkdocs materials using a custom home page
+
+[" if the theme.custom_dir configuration value is used in combination with an existing theme, the theme.custom_dir can be used to replace only specific parts of a built-in theme"](https://www.mkdocs.org/dev-guide/themes/)
+
+["built-in themes are implemented in a base.html file, which main.html extends"](https://www.mkdocs.org/dev-guide/themes/)
+
+### Create a custom theme
```
Comment (jreiberkyle) [inline design-docs/content-gsx-notes.md:19]: this seems nice, absolutely. My question is, where does a custom theme come in for priorities and what is the LOE?

Comment (jreiberkyle) [reply design-docs/content-gsx-notes.md:19]: concerns I have are 1) too custom means users have to orient to a new paradigm and 2) it's an additional chunk of code to maintain

File: docs/cli/cli-guide.md [hunk 13]
```diff
@@ -0,0 +1,179 @@
...
+```console
+$ planet auth init
+```
+
+To store the authentication information in an environment variable, e.g.
+for passing into a Docker instance:
+
+```console
+$ export PL_API_KEY=$(planet auth value)
+```
```
Comment (jreiberkyle) [inline docs/cli/cli-guide.md:19]: this seems like a good time to mention that the presence of the environment variable overrides changes to the stored auth with planet auth init

**Review by jreiberkyle — CHANGES_REQUESTED** [review]
Whew, nice work! I don't have time to review at a low level atm so I'll give high-level thoughts. My preference is to keep the name of the package as Planet SDK for Python (aka not change to *Toolkit) and to refer to the python api and cli aspects as CLI and API, respectively. I would prefer to see the the custom theme files to be inside the docs folder, if possible. I love the idea of the custom theme and am also somewhat hesitant to maintain code non-python code in this repo.

I couldn't look into it but there is a userguide.md - it may make sense to break that file up 
 and pull out the cli/api portions into their own homes and delete it, if it hasn't been done already,

**Pull request closed without merging**